sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -161,10 +161,10 @@ def get_estimated_size_and_observables(self, value, nested=False):
       return self.estimate_size(value, nested), []
 
 
-class DeterministicPickleCoderImpl(CoderImpl):
+class DeterministicCoderImpl(CoderImpl):
```

Comment:
I was thinking more in terms of not having a DeterministicCoderImpl, but just an Adapter/Decorator that converts any coder to one. But I don't have an entire picture of how coders and coder_impls interact, so not sure if this is possible.